src/js/components/DateInput/stories/Format.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, DateInput } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const Example = () => {
+  const [value, setValue] = React.useState('');
+  const onChange = event => {
+    const nextValue = event.target.value;
+    console.log('onChange', nextValue);
+    setValue(nextValue);
+  };
+  return (
+    <Grommet theme={grommet}>
+      <Box align="center" pad="large">
+        <Box width="medium">
+          <DateInput format="mm/dd/yyyy" value={value} onChange={onChange} />
```

Comment:
Food for thought, I noticed that each time we are using the `format` prop, the Calendar icon is not changing the cursor display from 'text' to 'pointer', I think 'pointer' makes more sense to me.

Reply:
That's because we're using the icon within MaskedInput. It is working consistently with https://storybook.grommet.io/?path=/story/textinput--icon

Reply:
Yes, I know. 
The reason I said its 'food for thought' is that I understand the complexity of this enhancement is placed in TextInput and not part of this PR.
But for me, it makes sense that if the TextInput uses a drop then the cursor on the icon should change to a pointer.

Reply:
got it